internal/handler/getUserFromContext_test.go

```diff
@@ -0,0 +1,47 @@
+package handler
+
+import (
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/dhis2-sre/im-user/pkg/model"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	"gorm.io/gorm"
+)
+
+func TestGetUserFromContext_Happy(t *testing.T) {
+	id := uint(0)
+	w := httptest.NewRecorder()
```

Comment:
I would inline vars that are only used once. This makes it easier to read the test. 

Can you get away with passing `nil`? I mean if we don't write to the response writer of an HTTP request we might. I would only add the setup to a test that is truly needed.